src/components/FilterPanel.jsx

```diff
@@ -70,12 +100,36 @@ export default function FilterPanel({ onClose }) {
             expandIcon={<ExpandMoreIcon />}
             aria-controls={name + '-content'}
             id={name + '-header'}>
-            <Typography>{name}</Typography>
+            <Grid container alignItems="center" justifyContent="space-between">
+              <Typography>{name}</Typography>
+              <Badge
+                badgeContent={filter.badge}
+                color="primary"
+                sx={{ mr: 2 }}
+              />
+            </Grid>
           </AccordionSummary>
 
-          <AccordionDetails sx={{ m: 1 }}>{filter}</AccordionDetails>
+          <AccordionDetails sx={{ m: 1 }}>{filter.comp}</AccordionDetails>
         </Accordion>
       ))}
+
+      <Toolbar
+        sx={{ justifyContent: { xs: 'center', md: 'space-between' }, mt: 2 }}>
+        <Button
+          variant="contained"
+          onClick={() => {
+            setQueryParams(filterVals);
+            onClose();
+          }}>
+          Apply Filters
+        </Button>
+        <Button
+          sx={{ display: { xs: 'none', md: 'block' } }}
+          onClick={resetFilters}>
+          Clear Filters
+        </Button>
```

Comment:
For consistency between mobile and desktop experiences I'd prefer to either keep this around for mobile for consistency OR remove this button entirely and make this bar appear on desktop:
<img width="372" alt="Screen Shot 2022-02-05 at 4 09 20 PM" src="https://user-images.githubusercontent.com/8023233/152663217-3a109328-c599-43ea-a7aa-0dcf610ef7a1.png">

I'm not fond of the idea of having two different reset buttons if they do the same thing (unless one is like "clear + apply" but the other is just "clear")

Reply:
went with the first option